pages/account/manage/event/[[...data]].js

```diff
@@ -79,7 +79,7 @@ export default function ManageEvent({ BASE_URL, event }) {
       url,
       date: { start: startDate, end: endDate },
       isVirtual,
-      price: { startingFrom: price },
+      price: { startingFrom: price || 0 },
```

Comment:
I would disagree that the price should be set to zero in the case of no price.  What if the price was unknown or simply not entered by the event submitter? I would suggest that we do not show price if it is unknown as this could be misleading or incorrect. wdyt?

Reply:
Oh good point - we can then display `unknown` if there is no value

Reply:
sorry for the late commit but i don't why mongodb not connection with docker compose, in the end i push the commit with git pod